.github/workflows/build_and_test_android.yml

```diff
@@ -106,14 +106,17 @@ jobs:
 
   test:
     # These physical devices are not scalable. Only run on postsubmit for now.
-    if: (! inputs.is-pr)
+    # if: (! inputs.is-pr)
     needs: cross_compile
     strategy:
       matrix:
-        # TODO(#9855): Add pixel-6-pro and moto-edge-x30.
         target:
           - device-name: pixel-4
-            label-exclude: vulkan
+            label-exclude: "vulkan"
+          - device-name: pixel-6-pro
+            label-exclude: "^requires-gpu-nvidia$"
+          - device-name: moto-edge-x30
+            label-exclude: "^requires-gpu-nvidia$"
```

Comment:
This is still queued: https://github.com/openxla/iree/actions/runs/4927131607/jobs/8804014322?pr=13441

Reply:
Fixed the configuration on the runner. The run is now finished
https://github.com/openxla/iree/actions/runs/4927131607/jobs/8804014322